### PR TITLE
Added initial database connection and api

### PR DESCRIPTION
**Pull request by jbevan4**
Co-authored-by: Stephen Thomson <stephen.thomson@madetech.com>

**What**  
Added initial database connections

**Why**  
So we can query our api and it hit the database

**Anything else?**

- A lot of the code added can be removed. I need to get this on master, click the button, the remove all the apis and buttons afterwards.

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -5,8 +5,9 @@ provider:
   runtime: nodejs12.x
   region: eu-west-2
   stage: ${opt:stage}
+  dbname: discretionaryBusinessGrantsDb
 #  environment:
-#    PLANS_TABLE: '${self:service}-${self:provider.stage}-plans'
+#    PLANS_TABLE: '${self:service}-${self:provider.stage}-plans'
```
Comment (StephenThomson) [inline serverless.yml:10]: We can remove this now

File: serverless.yml [hunk 2]
```diff
@@ -138,3 +152,12 @@ custom:
   aliases:
     staging: staging-discretionarybusinessgrants.hackney.gov.uk
     production: discretionarybusinessgrants.hackney.gov.uk
+
+  staging:
```
Comment (StephenThomson) [inline serverless.yml:156]: These should be under a label similar to aliases

Comment (jbevan4) [reply serverless.yml:156]: Got a quick minute to jump in a call? I'll show you why they're at this level.

Comment (jbevan4) [reply serverless.yml:156]: Ah do you mean under a subnet label?

**Pull request merged**